docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,40 @@
+# Sites
+
+## Overview Study
+
+**Sites** or **centres** are terms used interchangeably in LORIS.
+**Sites** are defined in the `psc` table of the database.
+
+LORIS' default schema defines the first psc (CenterID=1) as the
+Data Coordinating Center or "DCC".  This site is generally used as
```

Comment:
... This site is generally used **for** 

(for instead of as might flow better?)